src/table/handle-data.js

```diff
@@ -20,7 +20,7 @@ export function getColumnInfo(layout, colIndex) {
       isDim,
       width: 200,
       label: info.qFallbackTitle,
-      id: info.cId,
+      id: index,
```

Comment:
`colIndex` should work fine. All the key needs to be is a unique identifier in the current list. Not sure if necessarry, but I think a prefix would be nice, so `col-0` etc

Reply:
sure